PF-build.sh

```diff
@@ -110,74 +157,223 @@ if ! type zip > /dev/null; then
+		echo "As Python 2.x will not be maintained from 2020 please,"
+		echo "install it with the command $(tput setaf 2)'sudo apt-get install python3'."
+		echo "Check which version of Python3 has been installed using 'ls /usr/bin/python3*'"
+		echo "Use 'sudo ln -sf /usr/bin/python3.x /usr/bin/python' (where 'x' is your version number) to make it default.$(tput sgr0)"
+		exit 4
 	fi
 fi
-###End prepare bash enviroment
 
-BUILD_ENV="1.0.2"
+#### End prepare bash environment
+
+
+#### Set build environment 
+ARDUINO_ENV="1.8.5"
+BUILD_ENV="1.0.6"
+BOARD="PrusaResearchRambo"
+BOARD_VERSION="1.0.1"
+BOARD_URL="https://raw.githubusercontent.com/DRracer/Arduino_Boards/master/IDE_Board_Manager/package_prusa3d_index.json"
+BOARD_FILENAME="prusa3drambo"
+BOARD_FILE_URL="https://raw.githubusercontent.com/DRracer/Arduino_Boards/master/IDE_Board_Manager/prusa3drambo-1.0.1.tar.bz2"
+PF_BUILD_FILE_URL="https://github.com/3d-gussner/PF-build-env/releases/download/$BUILD_ENV/PF-build-env-$BUILD_ENV.zip"
+LIB="PrusaLibrary"
 SCRIPT_PATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
 # List few useful data
 echo
-echo "Script path:" $SCRIPT_PATH
-echo "OS         :" $OS
-echo "OS type    :" $OSTYPE
+echo "Script path :" $SCRIPT_PATH
+echo "OS          :" $OS
+echo "OS type     :" $OSTYPE
+echo ""
+echo "Ardunio IDE :" $ARDUINO_ENV
+echo "Build env   :" $BUILD_ENV
+echo "Board       :" $BOARD
+echo "Specific Lib:" $LIB
 echo ""
 
-#### Start prepare building
+#### Start prepare building environment
 
 #Check if build exists and creates it if not
 if [ ! -d "../build-env" ]; then
-    mkdir ../build-env || exit 2
+    mkdir ../build-env || exit 5
 fi
 
-cd ../build-env || exit 3
+cd ../build-env || exit 6
+BUILD_ENV_PATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
 # Check if PF-build-env-<version> exists and downloads + creates it if not
-# The build enviroment is based on the Arduino IDE 1.8.5 portal version with some changes
+# The build environment is based on the supported Arduino IDE portable version with some changes
 if [ ! -d "../PF-build-env-$BUILD_ENV" ]; then
 	echo "$(tput setaf 6)PF-build-env-$BUILD_ENV is missing ... creating it now for you$(tput sgr 0)"
 	mkdir ../PF-build-env-$BUILD_ENV
 	sleep 5
 fi
 
+# Download and extract supported Arduino IDE depending on OS
+# Windows
 if [ $OSTYPE == "msys" ]; then
-	if [ ! -f "PF-build-env-Win-$BUILD_ENV.zip" ]; then
-		echo "$(tput setaf 6)Downloading Windows build environment...$(tput setaf 2)"
+	if [ ! -f "arduino-$ARDUINO_ENV-windows.zip" ]; then
+		echo "$(tput setaf 6)Downloading Windows 32/64-bit Arduino IDE portable...$(tput setaf 2)"
 		sleep 2
-		wget https://github.com/3d-gussner/PF-build-env/releases/download/Win-$BUILD_ENV/PF-build-env-Win-$BUILD_ENV.zip || exit 4
-		#cp -f ../../PF-build-env/PF-build-env-Win-$BUILD_ENV.zip PF-build-env-Win-$BUILD_ENV.zip || exit4
+		wget https://downloads.arduino.cc/arduino-$ARDUINO_ENV-windows.zip || exit 7
+		echo "$(tput sgr 0)"
```

Comment:
Line duplicate?

Reply:
Don't understand...
`wget` is used 3 times:
- to download Arduino IDE 1.8.5 directly from Arduino web site
- to download Prusa Arduino boards definition
- to download Prusa specific library change in `avr6.xn` to support 2nd language

Reply:
I meant the line
234: echo "$(tput sgr 0)"
235: echo "$(tput sgr 0)"

Reply:
Changed